src/parser/jobs/rpr/modules/ArcaneCircle.tsx

```diff
@@ -20,28 +29,54 @@ export class ArcaneCircle extends RaidBuffWindow {
 
 		this.addEvaluator(new ExpectedActionsEvaluator({
 			expectedActions: [
-				{action: this.data.actions.COMMUNIO, expectedPerWindow: 2},
-				{action: this.data.actions.PLENTIFUL_HARVEST, expectedPerWindow: 1},
+				{
+					action: this.data.actions.COMMUNIO,
+					expectedPerWindow: 2,
+				},
+				{
+					action: this.data.actions.PLENTIFUL_HARVEST,
+					expectedPerWindow: 1,
+				},
 			],
 			suggestionIcon: this.data.actions.ARCANE_CIRCLE.icon,
 			suggestionContent: <Trans id="rpr.arcanecircle.suggestions.content">
-				Each <ActionLink action="ARCANE_CIRCLE"/> window should contain 2 uses of <ActionLink action="COMMUNIO"/> and a use
-				of <ActionLink action="PLENTIFUL_HARVEST"/>.
+				Each <ActionLink action="ARCANE_CIRCLE"/> window should contain 2 uses of <ActionLink action="COMMUNIO"/>
```

Comment:
I think a more wordy but also more explicit suggestion would be better. Theoretically this language should also be standardized across the site:

"Each {buff} window [after the opener] should contain N uses of {skill} [, M uses of {another_skill}, and O uses of {third_skill}.] The {buff} window in the opener should contain N uses of {skill} [, M uses of {another_skill}, and O uses of {third_skill}.]"

The purpose of this writing is to be informative/instructional, and not to sound necessarily like speech.